dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/RelationshipOperationParams.java

```diff
@@ -31,6 +31,7 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import org.hisp.dhis.tracker.imports.TrackerType;
```

Comment:
Are we adding a dependency from export to import or did it already exist? I mean the Class might also do as a discriminator. I am just worried we open the gates before thinking more about structure.

Not sure `TrackerType` is worth that, just yet. At least from the changes I am not sure 🤔

Reply:
I saw that and I actually forgot to move it outside of `imports` and under `tracker` package.
For me, `TrackerType` should be part of `tracker` package, it is not an import or export concept